CHANGELOG.asciidoc

```diff
@@ -3,6 +3,16 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.15.2]]
+=== Beats version 8.15.2
+https://github.com/elastic/beats/compare/v8.15.0\...v8.15.2[View commits]
```

Comment:
is this auto-generated?

Reply:
we had initially discussed on the version here- [PR](https://github.com/elastic/beats/pull/40461#discussion_r1762982165)